tasks/pipeline.py

```diff
@@ -860,7 +860,8 @@ def update_gitlab_config(file_path, image_tag, test_version):
     with open(file_path, "r") as gl:
         file_content = gl.readlines()
     gitlab_ci = yaml.safe_load("".join(file_content))
-    suffixes = [name for name in gitlab_ci["variables"].keys() if name.endswith("SUFFIX")]
+    # TEST_INFRA_DEFINITION_BUILDIMAGE label format differs from other buildimages
+    suffixes = [name for name in gitlab_ci["variables"].keys() if name.endswith("SUFFIX") and not name.startswith("TEST_INFRA_DEFINITION")]
```

Comment:
```suggestion
    suffixes = [name for name in gitlab_ci["variables"] if name.endswith("SUFFIX") and not name.startswith("TEST_INFRA_DEFINITION")]
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏More pythonic way of doing this, wdyt?